fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java

```diff
@@ -918,6 +915,30 @@ public void pageOrientation(String orientation) {
         this.pageOrientation = orientation;
     }
 
+    public void repeatRows(int startRow, int endRow) {
+        this.repeatingRows = new ArrayList<Integer>();
```

Comment:
Use a specific class to store a range. Like 
```
private static class RepeatRange { 
  final int start;
  final int end;
}
```

Reply:
made it a public class, since it also needs to be used outside of the Worksheet class (in the Workbook class)